content/guides/compute.md

```diff
@@ -4,3 +4,331 @@ title: Compute Engine documentation | ARGO
+
+######section: `[datastore-mapping]`
+This section contains various optional parameters used for correctly mapping results to expected datastore fieldnames
+- `e_map={fieldname1},{fieldname2}...,{fieldnameN}`  
+list for mapping group endpoint availability results to datastore fieldname conventions
+- `s_map={fieldname1},{fieldname2}...,{fieldnameN}`  
+list for mapping group service availability results to datastore fieldname conventions  
+- `n_eg={STRING}`
+endpoint group name type used in status detailed calculations
+- `n_gg={STRING}`
+group of groups name type used in status detailed calculations
+- `n_gg={STRING}`
+group of groups name type used in status detailed calculations
+- `n_alt={STRING}`
+mapping of alternative grouping parameter used in status detail calc.
+- `n_altf={STRING}`
+mapping of alternative grouping parameter used in status detail calc.
+- `service_dest={db_name}.{collection_name}`  
+destination for storing service a/r reports
+- `service_dest={db_name}.{collection_name}`  
+destination for storing endpoint grouped a/r reports
+- `sdetail_dest={db_name}.{collection_name}`  
+destination for storing endpoint grouped a/r reports
+
+#### Contents of /etc/ar-compute directory
+As mentioned above secondary configuration files used by the compute-engine are stored to the /etc/ar-compute directory. Here are files describing the set of status state types used in the monitoring engine, algorithmic operations of how to combine those states, availability profiles & configuration files for available jobs.
+
+##### Ops Files (per tenant)
+The ops files are json filetypes that are used to describe the available status types encountered in the monitoring enviroment of a tenant and also the availabe algorithmic operations available to use in status aggregations. The filename template used is very specific and is the following:
+`{TENANT_NAME}_ops.json`  
+for eg. if the tenant name is T1 the corresponding ops filename will be  
+`T1_ops.json`
+
+###### Contents of an ops file
+An ops file contains:
+- the list of available status types 
+- which status type is considered as default in missing circumstances
+- which status type is considered as default in downtime circumstances
+- which status type is considered as default in unknown circumstances
+- a list of available operations between statuses expressed in the form of truth tables
+ 
+The available status states in the metric enviroment are expressed in the "states" list as described below 
+```
+"states":
+  [
+    "OK",
+    "WARNING",
+    "UNKNOWN",
+    "MISSING",
+    "CRITICAL",
+    "DOWNTIME"
+  ]
+```
+
+Based on the availabe states declared in the "states" list are then declared default missing,unknown and downtime states for eg:
+
+```
+ "default_down": "DOWNTIME",
+ "default_missing": "MISSING",
+ "default_unknown": "UNKNOWN",
```

Comment:
explained in commit: https://github.com/kaggis/argodoc/commit/f4a15ac3584730c16c4b670cadceacf7a6ce1665